tests/functions_tests/test_maths.py

```diff
@@ -79,35 +80,51 @@ def test_output(self):
     {'op_name': 'Sub', 'ops': 'a - b'},
     {'op_name': 'Mul', 'ops': 'a * b'},
     {'op_name': 'Div', 'ops': 'a / b'},
-    {'op_name': 'MatMul',
+    {'op_name': 'MatMul_transa',
+     'ops': 'chainer.functions.matmul(a, b, transa=True)'},
+    {'op_name': 'MatMul_transb',
      'ops': 'chainer.functions.matmul(a, b, transb=True)'},
     {'op_name': 'Maximum', 'ops': 'chainer.functions.maximum(a, b)'},
     {'op_name': 'Minimum', 'ops': 'chainer.functions.minimum(a, b)'},
 )
 class TestBinaryMathOperators(ONNXModelTest):
 
-    def setUp(self):
+    def get_model(self):
         class Model(chainer.Chain):
 
             def __init__(self, ops):
                 super(Model, self).__init__()
                 self.ops = ops
 
             def __call__(self, a, b):
-                if not isinstance(a, chainer.Variable):
-                    a = chainer.Variable(a)
-                if not isinstance(b, chainer.Variable):
-                    b = chainer.Variable(b)
                 return eval(self.ops)
 
-        self.model = Model(self.ops)
-        a = chainer.Variable(input_generator.increasing(2, 3))
-        b = chainer.Variable(input_generator.nonzero_increasing(2, 3) * 0.3)
-        self.x = (a, b)
+        return Model(self.ops)
 
     def test_output(self):
-        name = self.op_name.lower()
-        self.expect(self.model, self.x, name=name)
+        test_cases = ['matrix', 'vector']
+        if not self.op_name.startswith('MatMul'):
```

Comment:
[optional] I guess it's OK to remove this check?

Reply:
Is this means we can skip scalar test? I don't have strong opinion but leave this test case to make sure.